dev/breeze/src/airflow_breeze/commands/kubernetes_commands.py

```diff
@@ -147,8 +148,7 @@ def kubernetes_group():
 K8S_CONFIGURE_CLUSTER_PROGRESS_REGEXP = r'.*airflow-python-[0-9.]+-v[0-9.].*'
 K8S_DEPLOY_PROGRESS_REGEXP = r'.*airflow-python-[0-9.]+-v[0-9.].*'
 K8S_TEST_PROGRESS_REGEXP = r'.*airflow-python-[0-9.]+-v[0-9.].*|^kubernetes_tests/.*'
-PERCENT_K8S_TEST_PROGRESS_REGEXP = r'^kubernetes_tests/.*\[[ \d*%]*\].*'
-K8S_SKIP_TRUNCATION_REGEXP = r'^kubernetes_tests/.*'
```

Comment:
I found better ways to truncate the output (I simply check the length of the output after stripping or ANSI Colours (standard textwrap does not truncate ANSI colors and treats ANSI colors as characters)